styles/theme.scss

```diff
@@ -0,0 +1,25 @@
+// these colors can be customized for each webstore instance. they are the default bootstrap theme colors and are used in bootstrap's theme_colors sass map, so updating these will also make sure that text overlaying areas with these background colors remains accessible. See https://getbootstrap.com/docs/5.0/customize/sass/#maps-and-loops for more details on how to add additional theme colors to this map. 
```

Comment:
can we move this file into "utils/theme"? remember to export it from "theme/index" if it's needed outside the "theme" folder.

Reply:
actually....it feels like we don't need these colors defined here AND in colors.js?? right? 

what do you think about putting this code into colors.js in place of lines 7-12 and deleting this file?